consensus/ibft/src/integration-test/java/org/hyperledger/besu/consensus/ibft/tests/SpuriousBehaviourTest.java

```diff
@@ -100,12 +100,14 @@ public void messageWithIllegalMessageCodeAreDiscardedAndDoNotPreventOngoingIbftO
   public void nonValidatorsCannotTriggerResponses() {
     final KeyPair nonValidatorKeys = KeyPair.generate();
```

Comment:
Could just use the NodeKey.generate here to then use do nodeKey.getPublicKey() to get the address for NodeParams